action-whitelist/README.md

```diff
@@ -0,0 +1,70 @@
+# action-whitelist
+Scan `.github/workflows/` for `.yml files` and loop through all workflows, checking their versions against a list of approved authors and versions contained in `approved.json`.
+
+The output is stored with the name `actions`, which can be retrieved in another action with `${{ steps.<step id>.outputs.actions }}`.
```

Comment:
Ultimately, we want the job to fail if anything is found. Preferably, the failure notice would be in the job log without having to download and open an attached artifact.

Reply:
Remove this part, as it was only for my local development and isnt relevant anymore. Everything is now only output in the job log